.cbmc-batch/jobs/aws_cryptosdk_decrypt_body/aws_cryptosdk_decrypt_body_harness.c

```diff
@@ -41,7 +41,8 @@ void aws_cryptosdk_decrypt_body_harness() {
 
     struct content_key *content_key;
 
-    uint8_t *tag = can_fail_malloc(props->tag_len);
+    /* Need to allocate tag_len bytes for writing the tag */
+    uint8_t *tag = malloc(props->tag_len);
```

Comment:
`can_fail_malloc()` followed by `assume != null` is better documentation.
also, how do we know its safe to assume nonnull?

Reply:
If the tag is NULL, then the assert in EVP_CIPHER_CTX_ctrl that taglen bytes can be written to buffer tag fails. I realize that there is no corresponding precondition on decrypt_body to ensure this though so I have gone ahead and added one. The analogous applied for encrypt_body.